go.mod

```diff
@@ -78,3 +75,7 @@ require (
 replace github.com/hashicorp/go-immutable-radix => github.com/tonistiigi/go-immutable-radix v0.0.0-20170803185627-826af9ccf0fe
 
 replace github.com/jaguilar/vt100 => github.com/tonistiigi/vt100 v0.0.0-20190402012908-ad4c4a574305
+
+replace github.com/containerd/containerd => github.com/containerd/containerd v1.3.1-0.20191217142032-9b5581cc9c5b
+
+replace github.com/docker/docker => github.com/docker/docker v1.4.2-0.20191210192822-1347481b9eb5
```

Comment:
The versions used in these `replace` lines doesn't match the commit-hash used in the `require` list above.